pygfx/geometries/_box.py

```diff
@@ -14,24 +14,43 @@ def box_geometry(
     height_segments=1,
     depth_segments=1,
 ):
-    """Create geometry respresenting a box.
-
-    The box has its center at the origin. The normals for each side all
-    point in the same direction; the corners are actually square.
-    Texture coordinates for each side run from 0 to 1.
-
-    Parameters:
-        width (int): The size in the x-dimension, default 1.
-        height (int): The size in the y-dimension, default 1.
-        depth (int): The size in the z-dimension, default 1.
-        width_segments (int): The number of segments to use in x.
-        height_segments (int): The number of segments to use in y.
-        depth_segments (int): The number of segments to use in z.
-    """
+    """Generate a box (rectangular cuboid).
+
+    Creates a box (a rectangular cuboid) of the given size that is centered
+    around the local frame's origin. Faces may be subdivided by specifying the
+    number of segments along each axis. This will result in all faces parallel to
+    the given axis to be evenly devided into the requested number of segments.
+
+    .. @almarklein: What does it mean for corners to be square?
+
+    The normals for each side all point in the same direction; the corners are
+    actually square. Texture coordinates for each side run from 0 to 1.
+
```

Comment:
Meh, let's drop this comment. The thought was that if you'd share the corner vertices, they'd appear oddly rounded. But this comments feels out of place now.
```suggestion
```